### PR TITLE
Add an -l option to switch https to http links

### PR DESCRIPTION
**Pull request by rallytime**
ping @cachedout and @s0undt3ch

### CODE REVIEW

**Comment by rallytime** [comment]
Lint errors are unrelated.

**Comment by s0undt3ch** [comment]
:+1:

But you'll need to rebase, conflicts...

File: bootstrap-salt.sh [hunk 1]
```diff
@@ -4474,7 +4485,7 @@ install_openbsd_deps() {
             # Let's download, since they were not provided, the default configuration files
             if [ ! -f "$_SALT_ETC_DIR/$fname" ] && [ ! -f "$_TEMP_CONFIG_DIR/$fname" ]; then
                 ftp -o "$_TEMP_CONFIG_DIR/$fname" \
-                    "https://raw.githubusercontent.com/saltstack/salt/develop/conf/$fname" || return 1
+                    "${HTTP_VAL}://raw.githubusercontent.com/saltstack/salt/develop/conf/$fname" || return 1
```
Comment (vutny) [inline bootstrap-salt.sh]: Is that really necessary for GitHub? Anyhow you'll get HTTP 301 code with `https://` URL.

``` console
$ curl -v http://raw.githubusercontent.com/saltstack/salt/develop/conf/master
*   Trying 23.235.39.133...
* Connected to raw.githubusercontent.com (23.235.39.133) port 80 (#0)
> GET /saltstack/salt/develop/conf/master HTTP/1.1
> Host: raw.githubusercontent.com
> User-Agent: curl/7.47.0
> Accept: */*
> 
< HTTP/1.1 301 Moved Permanently
< Server: Varnish
< Retry-After: 0
< Location: https://raw.githubusercontent.com/saltstack/salt/develop/conf/master
< Content-Length: 0
< Accept-Ranges: bytes
< Date: Tue, 12 Apr 2016 11:55:43 GMT
< Via: 1.1 varnish
< Connection: close
< X-Served-By: cache-atl6231-ATL
< X-Cache: HIT
< X-Cache-Hits: 0
< Access-Control-Allow-Origin: *
< Expires: Tue, 12 Apr 2016 12:00:43 GMT
< Vary: Authorization,Accept-Encoding
< 
* Closing connection 0
```

**Comment by vutny** [comment]
@rallytime I'm just wondering, maybe `-I` option would serve your needs?

```
    -I  If set, allow insecure connections while downloading any files. For
        example, pass '--no-check-certificate' to 'wget' or '--insecure' to
        'curl'
```

Thinking about an environment where 443 outgoing TCP port could be blocked... In any case it will not work with GitHub content downloading.

File: bootstrap-salt.sh (hunk 1 above)
Comment (rallytime) [inline bootstrap-salt.sh]: Oh, you're right. I'll change those references back to `https` instead of using `HTTP_VAL`.

**Comment by cachedout** [comment]
@paclat Will this serve your needs?

**Pull request merged**